js/util/struct_array.js

```diff
@@ -18,6 +19,10 @@ const viewTypes = {
     'Float64': Float64Array
 };
 
+/* eslint-disable no-undef */
+type ViewType = $Keys<typeof viewTypes>;
```

Comment:
It is a part of public API. See https://flowtype.org/docs/typeof.html and https://flowtype.org/docs/utility-types.html.